### ISSUE TITLE
Make ratings widget keyboard accessible

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons#9948]
### Describe the problem and steps to reproduce it:

- Visit https://amo.addons-dev.allizom.org/en-US/firefox/addon/adblock-plus/ 
- Using just the keyboard navigate to the ratings widget
- Select a rating star using the keyboard
- try and submit the rating

### What happened?

- Focus styles are shown and it's possible to tab to the different stars 
- The stars could change to yellow as they are focused along with the previous stars to provide a clearer visual indication of what submission would do.
- It's not possible to submit a rating without a click.

### What did you expect to happen?

- It should be possible to clearly navigate the widget and submit a rating without a mouse.

### Anything else we should know?

<img  alt="add-ons_for_firefox" src="https://cloud.githubusercontent.com/assets/1514/20171971/1113ef36-a72b-11e6-895e-43109131008d.png">
